bot/seasons/evergreen/nationday.py

```diff
@@ -0,0 +1,162 @@
+
+
+class NationDay(Cog):
+    """NationDay Cog that to help users get info on countries powered by the RESTCountries API."""
+
+    def __init__(self, bot: Bot):
+        self.bot = Bot
+        self.http_session: ClientSession = ClientSession()
+        with open(Path("bot/resources/evergreen/nationday/countries_by_day.json"), "r") as file:
+            self.by_day = json.load(file)
+
+        with open(Path("bot/resources/evergreen/nationday/iso_codes.json"), "r") as file:
+            self.iso_codes = json.load(file)
+
+    async def get_key(self, val: str) -> str:
+        """Get date for particular country, if present."""
+        for key, value in self.by_day.items():
+            if val in value:
+                return key
+        return None
+
+    async def get_specific_country(self, country: str) -> Tuple[List[Tuple[str, str]], str]:
+        """
+        Get Indepedence Day of given country.
+
+        Return Indepedence Day and page of information on country.
+        """
+        # Get date of specified country from dict
+        date = await self.get_key(country)
+        if date:
+            page, img = await self.get_country_info(self.http_session, country)
+            return [(page, img)], date
+        return [(None, None)], None
+
+    async def country_today(self) -> List[Tuple[str, str]]:
+        """
+        Get current day [Month & Day].
+
+        Return pages of info and flags of countries.
+        """
+        # Get current date [Month and day]
+        date = datetime.datetime.today()
+        month = date.strftime("%B")
+        day = date.day
+        today_date = f'{month} {day}'
+        today_date = today_date.lower()
+        try:
+            # Get list of countries
+            countries = self.by_day[today_date]
+        except KeyError as ke:
+            err_msg = f"**No countries have their independence day today.** {ke}"
+            logger.warning(err_msg)
+            return [(err_msg, white_flag)]
+        # Create pages
+        countries = list(countries.split(','))
+        pages = []
+        for country in countries:
+            page, img = await self.get_country_info(self.http_session, country)
+            pages.append((page, img))
+        return pages
+
+    async def get_country_info(self, client: ClientSession, country: str) -> Tuple[str, str]:
+        """
+        Create country information page using RESTCountries API.
+
+        Return page and flag image.
+        """
+        try:
+            iso_code = self.iso_codes[country]
+            if iso_code is None:
+                not_available_msg = f"Could not find information on this country."
+                logger.warning(not_available_msg)
+                return not_available_msg, white_flag
+            async with client.get(url+iso_code) as resp:
+                info = await resp.json()
+
+            country_info = ""
+            country_info += f"**Name**: {info['name']}\n\n"
```

Comment:
This looks better with just a single newline, doubling them ends up making quite a bit of whitespace in the embed.